drivers/crypto/crypto_mtls_shim.c

```diff
@@ -325,6 +402,25 @@ static int mtls_session_setup(struct device *dev, struct cipher_ctx *ctx,
 			ctx->ops.ccm_crypt_hndlr = mtls_ccm_decrypt_auth;
 		}
 		break;
+	case CRYPTO_CIPHER_MODE_GCM:
+#ifdef CONFIG_MBEDTLS_CIPHER_MODE_GCM_ENABLED
+		gcm_ctx = &mtls_sessions[ctx_idx].mtls_gcm;
+		mbedtls_gcm_init(gcm_ctx);
+		ret = mbedtls_gcm_setkey(gcm_ctx, MBEDTLS_CIPHER_ID_AES,
+					 ctx->key.bit_stream, ctx->keylen * 8U);
+		if (ret) {
+			LOG_ERR("Could not setup the key (%d)", ret);
```

Comment:
could you follow same style as in:
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/crypto/crypto_mtls_shim.c#L284
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/crypto/crypto_mtls_shim.c#L305
```suggestion
			LOG_ERR("AES_GCM: failed at setkey (%d)", ret);
```

Reply:
This has been changed. Thanks for spotting and the suggestion!